packages/rollup-plugin/src/index.ts

```diff
@@ -210,26 +229,27 @@ export function stylableRollupPlugin({
             };
 
             for (const moduleId of this.getModuleIds()) {
-                if (moduleId.endsWith(ST_CSS)) {
-                    modules.push({ depth: calcDepth(moduleId, context, [], cache), moduleId });
+                const { isStFile, isLoadableCssFile, path } = getLoadableModuleData(moduleId);
+                if (isStFile || isLoadableCssFile) {
+                    modules.push({ depth: calcDepth(moduleId, context, [], cache), path });
                 }
             }
 
             sortModulesByDepth(
                 modules,
                 (m) => m.depth,
-                (m) => m.moduleId,
+                (m) => m.path,
```

Comment:
why change?

Reply:
its just the inline type from a few lines back changing. there is a different between `moduleId` and `path` now that we add the query param to native CSS modules